### ISSUE TITLE
updating notes not working

### ISSUE THREAD

**Issue by Syberiyxx** [jonstodle/obsius-obsidian-plugin#4]
1. publish note in Obsius. 
2. open Obsius link in browser
3. add text
4. click "Update in Obsius"
5. refresh published note in browser
6. nothing has changed in published note

**Comment by Syberiyxx** [comment]
after ~12 hours, updates seem to have propogated

**Comment by jonstodle** [comment]
Yes. It's running on Cloudflare, which have quite agressive caching. I believe changes should be live afer approximately 4 hours. I hope to improve this in the future.
